packages/kbn-profiling-utils/common/functions.ts

```diff
@@ -101,7 +103,12 @@ export function createTopNFunctions({
 
     const lenStackTrace = stackTrace.FrameIDs.length;
 
-    for (let i = 0; i < lenStackTrace; i++) {
+    // Error frames only appear as first frame in a stacktrace.
+    const start =
+      // eslint-disable-next-line no-bitwise
+      !showErrorFrames && lenStackTrace > 0 && (stackTrace.Types[0] & 0x80) !== 0 ? 1 : 0;
```

Comment:
Can you define a const for the `0x80` value? This will avoid having to add the eslint skip(`// eslint-disable-next-line no-bitwise`) everywhere.

Reply:
Hm, the IDE still doesn't like it.